lib/helpers/umzug-helper.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+var args = require("yargs").argv;
+var path = require("path");
+var _    = require("lodash");
+
+module.exports = {
+  getStorageMethod: function () {
+    return args.storage || "sequelize";
```

Comment:
This function will return the content of the config file https://github.com/sequelize/cli/blob/master/lib/helpers/config-helper.js#L63